js/everything.js

```diff
@@ -516,7 +538,7 @@ $(function() {
                             window.location.href = makeSearchURL(formValue);
                         }
                         else {
-                            window.location.href = makeListenURL(videoID);
+                            window.location.href = makeListenURL(videoID, formValueTime);
```

Comment:
If `formValueTime` is set to an empty array from the regex match above (ie: no matches) we'll pass an empty array to `makeListenURL()` which is evaluated to a truthy value. I believe this means the URL will end up being something like `https://zenplayer.audio/?v=foo&t=` if no `t` param was passed  in.